tests/query_runner/test_yandex_disk.py

```diff
@@ -0,0 +1,45 @@
+from io import BytesIO
+from unittest import TestCase
+
+import pandas as pd
+
+from redash.query_runner.yandex_disk import EXTENSIONS_READERS
+
+test_df = pd.DataFrame(
+    [
+        {"id": 1, "name": "Alice", "age": 20},
+        {"id": 2, "name": "Bob", "age": 21},
+        {"id": 3, "name": "Charlie", "age": 22},
+        {"id": 4, "name": "Dave", "age": 23},
+        {"id": 5, "name": "Eve", "age": 24},
+    ]
+)
+
+test_token = "AAAAQAA"
+
+
+class TestYandexDisk(TestCase):
+    def test_xlsx(self):
```

Comment:
These tests are great, but we only have ~30% coverage of the new query runner. Could we add some more tests to have >=90%?